lib/injection_container.dart

```diff
@@ -15,8 +17,16 @@ void registerDependencies() {
   // Repositories
   sl.registerLazySingleton<ProductsRepository>(
       () => ProductsRepositoryImpl(sl()));
+      
 
   // Cubits
   sl.registerFactory(() => ProductsCubit(sl()));
   sl.registerFactory(() => CartCubit());
+
+  sl.registerLazySingleton<ProfileRepository>(
```

Comment:
Please put these statements in their corresponding places. You see the comments?

Reply:
i.e., the cubit dependency should be registered under the cubits section and the repository dependency should be registered under the repository section.